### DISCUSSION TITLE
February Node Earnings

### DISCUSSION THREAD

**Discussion by DiegoRBaquero** [filecoin-project/saturn-node#279] [Announcements]
# Top Countries

| Country| Nodes| Total FIL Earned | FIL per node
|---|---|---|---
Egypt|1|80|80
Guatemala|4|213|53
Saudi Arabia|3|143|48
Denmark|2|91|45
Costa Rica|4|181|45
Romania|6|248|41
Puerto Rico|6|237|39
Ecuador|10|359|36
Cambodia|1|34|34
Hungary|6|201|33
Malaysia|19|516|27
Moldova|4|92|23
Czech Republic|3|66|22
Ukraine|5|107|21
Peru|72|1354|19
Austria|16|289|18
Switzerland|15|275|18
Thailand|5|81|16
Brazil|62|911|15
Spain|178|2317|13
Turkey|21|253|12
Canada|168|1939|12
Lithuania|7|87|12
Norway|16|172|11
Argentina|51|489|10
United Kingdom|168|1749|10
Mexico|151|1482|10
Nigeria|9|80|9
Luxembourg|12|94|8
Chile|69|562|8
Colombia|84|666|8
Singapore|314|2115|7
Finland|26|185|7
New Zealand|8|58|7
Poland|23|166|7
Portugal|18|130|7
France|143|952|7
Germany|123|747|6
Israel|27|150|6
Italy|28|163|6
Australia|104|622|6
Bulgaria|53|317|6
Indonesia|431|2714|6
South Africa|47|277|6
Bahrain|28|138|5
Netherlands|92|445|5
Ireland|37|167|5
Hong Kong|88|399|5
United Arab Emirates|98|519|5
Japan|179|740|4
South Korea|206|743|4
Sweden|33|135|4
United States|1645|7282|4
Russia|41|173|4
Taiwan|16|54|3
India|421|1468|3
Belgium|141|373|3
Latvia|7|17|2

**Comment by zkaxin** [comment]
Is there any current data？

**Reply by 31z4** [reply]
Hi @mydoto. I haven't seen any official stats published since July. But I've done my own analysis based on historical data gathered since July using the same APIs that the official [Saturn Node Dashboard](https://dashboard.saturn.tech/stats) uses.

Here's the link to September earnings and traffic by country: https://31z4.github.io/saturn-observatory/?year=2023&month=09#earnings-and-traffic-by-country and here's the screenshot from that page:

<img width="948" alt="image" src="https://github.com/filecoin-saturn/L1-node/assets/3657959/03050206-db1e-4ed5-9fe5-d0b034763922">

Note that to make the chart readable and fit it on the screen not every country name is displayed. To see all countries follow the link above.

**Once again, this analytics is not official and may not be 100% accurate**.

**Reply by zkaxin** [reply]
We ran a few nodes, but all were beyond our means, not sure if there was a test compensation reward
